app/src/main/java/org/dasfoo/rover/android/client/grpc/task/GrpcConnection.java

```diff
@@ -88,7 +88,7 @@ public final boolean shutDownConnection() {
             return Boolean.TRUE;
         } catch (InterruptedException e) {
             if (BuildConfig.DEBUG) {
```

Comment:
We might need to invent something here: [Log](https://developer.android.com/reference/android/util/Log.html) (see **Tip 2**) suggests to filter out the log, but `BuildConfig.DEBUG` doesn't seem like a good signal anymore.